src/ArborX_BruteForce.hpp

```diff
@@ -63,23 +63,23 @@ class BasicBruteForce
   void query(ExecutionSpace const &space, Predicates const &predicates,
              Callback const &callback, Ignore = Ignore()) const;
 
-  template <typename ExecutionSpace, typename Predicates,
+  template <typename ExecutionSpace, typename UserPredicates,
```

Comment:
I changed the name only here because we will wrap the provided predicates in `AccessValues`. In other places, where no wrapping is done, it does not matter.

I don't feel strongly either way.